File: Kotlin(android_app)/app/java/jewelrystore/models/Order.kt
package com.sysoliatina.jewelrystore.models

import com.google.gson.annotations.SerializedName
import com.sysoliatina.jewelrystore.models.enums.OrderStatus

data class Order(
    val id: Int,
    var status: OrderStatus,
    @SerializedName("user_email") val userEmail: String,
    @SerializedName("user_name") val userName: String,
    val products: List<Product>,
    @SerializedName("user_id") val userId: Int,
    @SerializedName("order_date") val orderDate: String,
    @SerializedName("order_price") val orderPrice: Int,
)

File: Python(create_db)/models/order.py
from typing import List

from pydantic import BaseModel
from sqlite3 import Date

from enums.order_status import OrderStatus
from models.product import ProductInOrder
from models.tables import Product


class Order(BaseModel):
    id: int
    user_id: int
    user_email: str
    user_name: str
    order_date: Date
    order_price: int
    status: OrderStatus
    products: List[ProductInOrder] = []

    class Config:
        orm_mode = True